src/crypto/public_key/rsa.js

```diff
@@ -140,12 +140,33 @@ function RSA() {
     // Native RSA keygen using Web Crypto
     //
 
+    function convertKeyOperation(keyop, errmsg) {
+      return new Promise(function(resolve, reject) {
+        keyop.onerror = function (err) { 
+          reject(new Error(errmsg));
+        }
+        keyop.oncomplete = function (e) {
+          resolve(e.target.result);
+        }
+      });
+    }
+
```

Comment:
Sorry to nitpick. But perhaps it would be cleaner to add this function below https://github.com/openpgpjs/openpgpjs/pull/296/files#diff-aed44c6e8be49da021ed00cc64d9611eL183